src/components/pager/pager.spec.tsx

```diff
@@ -405,6 +405,46 @@ describe("Pager", () => {
 
       expect(wrapper.find(StyledPagerSummary).text()).toBe("");
     });
+
+    it("renders a label correctly-linked to the select when 'showPageSizeLabelBefore' is true, and does not add an aria-label", () => {
```

Comment:
**nitpick (non-blocking):** Could these two `its` be changed to an `it.each`? As far as I can tell, these tests are identical, apart from `showPageSizeLabelBefore` and `showPageSizeLabelAfter` having opposite truth\false values. This could help condense the file, I am aware `it.each`s are less readable at times, but when there will be a small amount of parameters, like here, it could be worth doing.

Reply:
that's another good point, thanks. In my head these were different tests because the label referenced by `selectLabel` is actually in a different place in the DOM in the two cases - but that doesn't actually change how the tests are structured or what they're fundamentally testing. I agree with you and will wrap them in an `it.each`.